src/test/java/seedu/address/model/entity/body/BodyIdentificationNumberTest.java

```diff
@@ -0,0 +1,73 @@
+package seedu.address.model.entity.body;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class BodyIdentificationNumberTest {
```

Comment:
Similar for this test class!

Reply:
Thanks for the reminder! Moved to main/unused.